gui/qt/contact_list.py

```diff
@@ -63,12 +64,22 @@ def import_contacts(self):
             self.parent.show_message(str(e))
         self.on_update()
 
+    def export_contacts(self):
+        filename = self.parent.getSaveFileName(_("Select file to save your contacts"), 'electrum_contacts.json', "*.json")
+        if  not filename:
+            return
+        try:
+            self.parent.contacts.export_file(filename)
+        except (IOError, os.error) as reason:
+            self.parent.show_critical(_("Electrum was unable to export your contacts.") + "\n" + str(reason))
```

Comment:
btw you can add a "success" pop-up as you originally did, I think that's a good idea. A clean way to do it is with a `try-except-else`, in the `else` branch. If you do, add that to that importing too. :P
(deleted the other comment regarding this, it was at the wrong pos.)